### DISCUSSION TITLE
Janusgraph 0.6.1 .profile() results getting truncated

### DISCUSSION THREAD

**Discussion by rkkris75** [JanusGraph/janusgraph#3702] [Q&A]
Hi there,

  Is there any config property or setting to avoid the .profile() o/p results from getting truncated? I did set up "query.max-profile-lines = <num of lines>" In the janusgraph-inmemory.properties file, but didn't help. In the below example, if you notice it is getting truncated after "Product_Fa..." to be precise.

Example:

HasStep([Source_System_Name.eq(CDF), Product_Fa... 0.329 0.17

**Comment by hadoopmarc** [comment]
The truncation stems from  the toString() method of:
https://github.com/apache/tinkerpop/blob/3.5.3/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/DefaultTraversalMetrics.java
I guess you worked in the gremlin console, which prints the objects returned implicitly calling toString(). You can just do:
```
    metrics = g.V().profile().next()
    metrics.getClass()
```
Now you can pass metrics to your own print function.

**Reply by rkkris75** [reply]
Thanks. Will try it out.
